### PR TITLE
Fix package data and scan data saving issue when exist bulk data

### PR DESCRIPTION
**Pull request by hanchuntao**
[OLCS-254](https://issues.redhat.com/browse/OLCS-254)

Fixes OLCS-254

### CODE REVIEW

**Comment by hanchuntao** [comment]
@rhyw @qduanmu @huiwangredhat PTAL

File: openlcs/packages/views.py [hunk 1]
```diff
@@ -783,6 +786,15 @@ class PackageImportTransactionView(APIView, PackageImportTransactionMixin):
             "nvr": "rhel8.0",
```
Comment (rhyw) [inline openlcs/packages/views.py:786]: `package` no longer exists. I guess L785-L789 could be removed.

File: openlcs/packages/mixins.py [hunk 2]
```diff
@@ -254,20 +200,64 @@ def save_scan_result(self, **kwargs):
...
+                            set(file_ids).difference(license_file_ids))
+                        with transaction.atomic():
+                            # Exist same files in different paths.
+                            self.save_file_license_scan(
+                                new_file_ids, license_detector)
+                            self.save_license_detections(
+                                path_file_dict, data, license_detector)
+                            self.update_scan_flag(source, "license_scan")
+                    except IntegrityError as err:
+                        if i == max_retries - 1:
```
Comment (rhyw) [inline openlcs/packages/mixins.py:222]: nitpick: suppose `SAVE_DATA_MAX_RETRIES=2`, this "==" means the forloop runs only once.

**Comment by rhyw** [comment]
Generally LGTM. much easlier to read than that of using retry decorators.

File: openlcs/packages/mixins.py (hunk 2 above)
Comment (hanchuntao) [reply openlcs/packages/mixins.py:222]: `i` will 0, 1 I think.

File: openlcs/packages/views.py (hunk 1 above)
Comment (hanchuntao) [reply openlcs/packages/views.py]: Removed, thanks

File: openlcs/packages/mixins.py (hunk 2 above)
Comment (rhyw) [reply openlcs/packages/mixins.py:222]: you want to retry twice in case MAX_RETRIES=2

Comment (hanchuntao) [reply openlcs/packages/mixins.py:234]: Yes, retry twice, the same as setting `MAX_RETRIES`

**Pull request merged**